### PR TITLE
Minify all.js and all.css

### PR DESCRIPTION
**Pull request by Androbin**
Towards #130

### CODE REVIEW

**Comment by sushain97** [comment]
Hm, Python libraries aren't exactly light in terms of dependencies either. However, I think this is an OK solution if and only if an individual without the libraries attempts to compile the app, it still works, it's just not minified. If you could make those changes and modify the README this looks good to me.

**Comment by Androbin** [comment]
If a dependency is not installed, minification will just skip and print a hint.
I have updated the README section on compression accordingly.

**Comment by sushain97** [comment]
I believe at least `csscompressor` has a built in command line version that would be preferable.

Also, please modify the Docker configs to also install the new dependencies.

**Comment by Androbin** [comment]
Okay, I have updated the `Makefile` and `Dockerfile` accordingly.

**Comment by sushain97** [comment]
This seems to fully break the app:

image

image

**Comment by sushain97** [comment]
Also, please don't force push. I made some commits that you should pull down that got overwritten last time.

File: README.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ Any changes to the local version of Html-tools will trigger a `make`.
 
 For production usage, remember to
 
-1. Build with `jsmin` and `csscompressor` installed through `pip`
-2. Enable gzip compression on your server
+1. Build with `jsmin` and `csscompressor` installed through `pip`.
+1. Enable gzip compression on your server.
```
Comment (Androbin) [inline README.md]: Why?

Comment (sushain97) [reply README.md]: We've had this conversation before. It makes diffs smaller if you need to insert something.

**Comment by Androbin** [comment]
@sushain97 I can't reproduce that error on my machine, neither with nor without `jsmin` installed. I have even put that file through multiple syntax checkers. Could you check if it's a problem on your side?

**Comment by sushain97** [comment]
@Androbin can you post updated versions of the numbers on #130?

**Comment by Androbin** [comment]
Do the minified versions already get shipped?

**Pull request merged**